catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/model/Search.java

```diff
@@ -102,6 +102,8 @@
 
     public static final String CACHED = "cached";
 
+    private static final String INTERNAL_LOCAL_RESOURCE = "internal.local-resource";
```

Comment:
This const is redefined in multiple places. Is there anywhere reasonable it could be made public and shared instead?